Source/m_misc.c

```diff
@@ -1302,7 +1302,7 @@ default_t defaults[] = {
     NULL, NULL,
     {0}, {UL,UL}, input, ss_keys, wad_no,
     "key to take a screenshot (devparm independent)",
-    input_screenshot, { {input_type_key, '*'} }
+    input_screenshot, { {input_type_key, KEYD_PRTSCR} }
```

Comment:
I don't know what `*` is supposed to mean? Doesn't work in DOSBox either.

Reply:
That'd be Shift+8 on a US keyboard, strange.

Reply:
Maybe it's a problem with my laptop keyboard, but `*` doesn't work. I think PrtScr is better anyway.

Reply:
Definitely!